src/App/Fossa/Container/Sources/Podman.hs

```diff
@@ -0,0 +1,60 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module App.Fossa.Container.Sources.Podman (analyzeFromPodman, podmanInspectImage) where
+
+import App.Fossa.Container.Sources.DockerArchive (analyzeFromDockerArchive)
+import Container.Types (ContainerScan)
+import Control.Carrier.Lift (Lift)
+import Control.Effect.Debug (Debug, Has)
+import Control.Effect.Diagnostics (Diagnostics)
+import Control.Effect.Path (withSystemTempDir)
+import Control.Effect.Telemetry (Telemetry)
+import Control.Monad (void)
+import Data.String.Conversion (ToText (toText), toString)
+import Data.Text (Text)
+import Effect.Exec (AllowErr (Never), Command (..), Exec, execThrow')
+import Effect.Logger (Logger, logInfo, pretty)
+import Effect.ReadFS (ReadFS)
+import Path (Abs, File, Path, mkRelFile, toFilePath, (</>))
+
+-- | Inspects a image, if image does not exist throws an error.
+-- Refer to: https://docs.podman.io/en/latest/markdown/podman-image-inspect.1.html
+podmanInspectImage :: Text -> Command
+podmanInspectImage img =
+  Command
+    { cmdName = "podman"
+    , cmdArgs = ["image", "inspect", img]
+    , cmdAllowErr = Never
+    }
+
+-- | Saves container image to a location in docker archive (tarball) format.
+-- Refer to: https://docs.podman.io/en/latest/markdown/podman-save.1.html
+podmanExtractImage :: Text -> Path Abs File -> Command
```

Comment:
We don't need to talk to socket over ssh, we can use the executable itself...

![image](https://user-images.githubusercontent.com/86321858/185265171-1735d988-012e-422e-b0fd-5c776ea67b9d.png)